ui/components/TracePage/TracePage.tsx

```diff
@@ -291,15 +318,21 @@ const TreeProvider = ({ traceId, children }: { traceId: string; children: ReactN
           if (id !== rootId && !isModelCall(calls[id]))
             setExpandedById(current => ({ ...current, [id]: expanded }));
         },
+        setExpandedById,
+        highlighted,
+        setHighlighted,
         getFocussed,
+        hideOthers,
+        setHideOthers,
+        isVisible,
```

Comment:
This context is getting seriously out of hand. We should probably do something about it this month